backend/cen2.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS CENKeys;
```

Comment:
What SQL sub-dialect is this targeting? Can we be explicit about that either in the folder path or the file itself?

Reply:
No idea, this was the syntax/semantics in the original cen.sql. If you know how to do what you are suggesting, please do it. I can't really add more context than this.
(I think it is mysql specifically, no knowledge of whether that dialect is impacted by versions of mysql. It worked on mysql 8 on my Mac, FYI.)